atf_core/scripts/analyser.py

```diff
@@ -178,18 +181,15 @@ def aggregate_results(self, atf_result):
                     for tl_test in tl_tests.keys():
                         #print "    tl_test=", tl_test
                         metric_result = MetricResult()
-                        started = True
-                        finished = True
+                        status = TestblockStatus.SUCCEEDED
                         groundtruth_result = True
```

Comment:
`groundtruth_result` should not be boolean - but also use the new enum....there are still quite some boolean `groundtruth_result` appearances in this file

Reply:
fixed in https://github.com/floweisshardt/atf/pull/150/commits/24ba26a2b83488bb61a08b9d2a9447f3da58d4ff